modules/services/window-managers/river.nix

```diff
@@ -180,28 +191,52 @@ in {
       ### SHELL VARIABLES ###
       ${config.lib.shell.exportAll cfg.extraSessionVariables}
 
+    '' + (lib.optionalString cfg.systemd.enable ''
```

Comment:
I moved systemd activation before configuration because we want `graphical-session.target` services to be started before `settings.spawm` apps are.